core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
IIRC these aren't abstract in the base, they have a value assigned. Thus both can be vals. Having the override here seems super superfluous but it shouldn't hurt anything.